.travis.yml

```diff
@@ -21,13 +21,16 @@ cache:
         - $HOME/.composer/cache
 
 before_install:
+    - mv $HOME/.phpenv/versions/$(phpenv version-name)/etc/conf.d/xdebug.ini $HOME/xdebug.ini || return 0
     - 'if [ "$SYMFONY_VERSION" != "" ]; then sed -i "s/\"symfony\/\([^\"]*\)\": \"[^\"]*\"/\"symfony\/\1\": \"$SYMFONY_VERSION\"/g" composer.json; fi'
 
 install:
+    - travis_retry composer global require hirak/prestissimo
```

Comment:
`travis_retry` is good for long with no output jobs.

This command would not take more than 10 minutes to execute.

Plus, `travis_retry` drop the original output of the command, harder to read.

This is as you want, but I suggest you to remove this. :+1: